src/adzerk/boot_reload.clj

```diff
@@ -82,9 +85,10 @@
         src  (tmp-dir!)
         tmp  (tmp-dir!)
         prev (atom nil)
-        out  (doto (io/file src "adzerk" "boot_reload.cljs") io/make-parents)]
+        out  (doto (io/file src "adzerk" "boot_reload.cljs") io/make-parents)
+        url  (start-server @pod {:ip ip :port port} domain)]
```

Comment:
Instead of new argument, we could just use the existing map and add domain to that.